app/build.gradle

```diff
@@ -48,6 +48,27 @@ android {
         }
     }
 
+    flavorDimensions "stage"
+    productFlavors {
+        production {
+            applicationId "co.nimblehq"
+        }
+
+        staging {
+            applicationId "co.nimblehq.staging"
+        }
+
+        development {
```

Comment:
Should be rename development -> develop 😃

Reply:
@Wadeewee we use noun here ;) `production` -> `development`